src/shared/infra/http/app.spec.ts

```
@@ -0,0 +1,163 @@
+          multiverse_designation: 'Earth-616',
+          entries: [
+            { id: 1, title: 'Iron Man', chronology_order: 1, type: 'movie' },
+          ],
+        },
+      ]),
+    );
+
+    const response = await request(app).get('/api/v1/timeline');
+
+    expect(response.status).toBe(200);
+    expect(response.body[0].entries[0]._links.self.href).toContain(
```

ESLint

[error] Unexpected dangling '_' in '_links'.

(no-underscore-dangle)